### PR TITLE
bloomstack-accdemy-fixes

### PR DESCRIPTION
**Pull request by meshramsaroj**
**Task Link:** https://app.asana.com/0/1194051119161697/1200693102541898/f
**Gif:**
updated-sidebar-with-details

### CODE REVIEW

File: erpnext/www/lms/course.html [hunk 1]
```diff
@@ -4,143 +4,138 @@
...
-		}
-		.continue-learning-btn:hover {
-			background-color: #fff;
-			color: #76c2ba;
-		}
-	</style>
+<style>
+	.image-body {
+		align-self: center;
+		color: #d1d8dd;
```
Comment (hrwX) [inline erpnext/www/lms/course.html]: - Use brand primary colour
- Not sticking to the default colour palette will result into bad UX

**Review by hrwX — CHANGES_REQUESTED** [review]

**Comment by nehasacher143** [comment]
@meshramsaroj Please add a mobile view GIF/Screenshot as well

**Comment by meshramsaroj** [comment]
> @meshramsaroj Please add a mobile view GIF/Screenshot as well

currently, I'm only working on desktop view. so there is no mobile view for now

**Comment by meshramsaroj** [comment]
I am closing this PR and creating a new PR with sorted code
@neilLasrado  @nehasacher143

**Pull request closed without merging**